first commit

thisisha opened the pull request.

jungbug commented on index.html:
```diff
@@ -1,9 +1,84 @@
+            justify-content: center;
+            align-items: center;
+            flex-direction: column; /* 내용을 수직으로 정렬 */
+            height: 100vh; /* 화면 전체 높이로 설정하여 수직 중앙 정렬 */
+            margin: 0; /* body의 기본 margin 제거 */
+        }
+        header {
+            background-color: lightgrey; /* 상단바 배경색을 회색으로 설정 */
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            padding: 10px;
+            width: 100%; /* 상단바 너비를 화면 전체로 설정 */
+            position: fixed; /* 페이지를 스크롤해도 상단바가 고정되도록 설정 */
+            top: 0; /* 상단바를 화면 상단에 고정 */
+        }
+        .title {
+            flex-grow: 1;
+        }
+        nav {
+            margin-left: 20px;
+        }
+        .profile {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            text-align: center; /* 텍스트 중앙 정렬 */
+            width: 80%; /* 내용의 너비를 설정 */
+            margin-top: 100px; /* 상단바와의 간격 조정 */
+        }
+        .profile-picture img {
+            width: 25%; /* 사진 크기를 4배 줄임 */
+            height: auto; /* 비율 유지 */
+            margin-right: 5px; /* 사진과 내용 사이 여백 */
+        }
+        .profile-details {
+            width: 50%; /* 내용의 너비를 설정 */
+            text-align: left; /* 자기소개 내용을 사진의 우측에 위치 */
+        }
+    </style>
```
이런식으로 작성하는것도 파일에 이상은 없지만 따로 css 폴더를 만들어서 그 안에 css 파일들을 만들어서 관리하시면 나중에 유지보수 하기가 편해요!

jungbug commented on index.html:
```diff
@@ -1,9 +1,84 @@
+        </section>
+    </main>
+
+    <footer>
+        <!-- 푸터 내용을 추가하세요 -->
+    </footer>
```
footer는 사용하지 않으니까 지워도 될거 같아요!

jungbug commented on pages/goals.html:
```diff
@@ -0,0 +1,111 @@
+        nav a {
+            margin-right: 20px;
+            text-decoration: none;
+            color: black;
+        }
+        .blocks {
+            display: flex; /* 블록들을 가로로 나란히 배치 */
+            justify-content: space-around; /* 블록들을 가로 방향으로 고르게 배치 */
+            width: 100%; /* 블록들을 가로로 꽉 차게 설정 */
+            margin-top: 50px; /* 블록과 상단바 사이 여백 설정 */
+        }
+        .block {
+            width: 300px; /* 블록의 너비를 설정 */
+            padding: 20px; /* 내용과 테두리 사이 여백 설정 */
+            background-color: white; /* 블록 배경색을 흰색으로 설정 */
+            text-align: center; /* 텍스트 중앙 정렬 */
+        }
+        .block + .block {
+            margin-left: 20px; /* 두 번째 블록부터 왼쪽 여백 설정 */
+        }
+        .block h2 {
+            background-color: darkgrey; /* 제목 배경색을 더 진한 회색으로 설정 */
+            color: white; /* 제목 글자색을 흰색으로 설정 */
+            padding: 10px; /* 제목의 위아래 여백 설정 */
+            margin: 0; /* 제목의 기본 margin 제거 */
+        }
+        .block p {
+            padding: 20px; /* 내용과 테두리 사이 여백 설정 */
+            background-color: lightgrey; /* 내용 배경색을 더 연한 회색으로 설정 */
+            color: black; /* 내용 글자색을 검은색으로 설정 */
+            margin: 0; /* 내용의 기본 margin 제거 */
+        }
+        .block-content {
+            background-color: lightgrey; /* 내용 배경색을 더 연한 회색으로 설정 */
+            padding: 20px; /* 내용과 테두리 사이 여백 설정 */
+            margin-top: 0px; /* 내용 위 여백 설정 */
+            margin-bottom: 30px; /* 내용 아래 여백 설정 */
+            text-align: center; /* 텍스트 중앙 정렬 */
+        }
+    </style>
```
여기도 마찬가지로 css 폴더 안에 파일 만들어서 관리하면 좋을거 같아요!

jungbug commented on pages/goals.html:
```diff
@@ -0,0 +1,111 @@
+<body>
+    <header>
+        <h1 class="title">[202001935] 이건하 | Goals</h1>
+        <nav>
+            <a href="index.html">About me</a>
+            <a href="pages/hobby.html">Hobby</a>
+            <a href="pages/goals.html">Goals</a>
+        </nav>
```
여기부분에서 경로가 잘못된거 같아요!

```html
        <nav>
            <a href="../index.html">About me</a>
            <a href="hobby.html">Hobby</a>
            <a href="goals.html">Goals</a>
        </nav>
```
이런식으로 바꿔주시면 아마 잘 들어가질 거에요!

jungbug commented on pages/hobby.html:
```diff
@@ -0,0 +1,66 @@
+<body>
+    <header>
+        <h1 class="title">[202001935] 이건하 | Hobby</h1>
+        <nav>
+            <a href="index.html">About me</a>
+            <a href="pages/hobby.html">Hobby</a>
+            <a href="pages/goals.html">Goals</a>
+        </nav>
```
여기 부분도 수정하시면 잘 들어가질거에요!

```html
        <nav>
            <a href="../index.html">About me</a>
            <a href="hobby.html">Hobby</a>
            <a href="goals.html">Goals</a>
        </nav>
```

jungbug reviewed (approved):
건하님!

1주차 미션하시느라 수고 많으셨어요!

우선 코드 부분에서는 따로 수정할게 없는거 같아요 👍
 시멘틱 태그들을 잘 사용해 주셨어요!
다만 코드에서 경로가 올바르지 않게 설정되어있는데 수정하시면 아마 버튼 누르면 잘 들어가질 거에요

또한 깃허브 사용에 관해서는 모르시면 저한테 연락 주시면 제가 도와드리겠습니다!
간단하게 push 하는 방법 알려드릴게요

---
```
git add . 
git commit -m "하고 싶은 말"
git push origin thisisha
```
이런식으로 작성 해주시면 되요!
자세한 내용은 [미션 진행 가이드](https://www.notion.so/f0571981555d4509839b9db8d5382162?pvs=21) 여기를 참고해주세요

thisisha commented:
네 확인했습니다! 감사합니다.
오늘도 좋은 하루 되세요 :)


-----Original Message-----
From: "JUNG SUNG ***@***.***>
To: ***@***.***>;
Cc: ***@***.***>; ***@***.***>;
Sent: 2024-02-19 (월) 02:00:57 (GMT+09:00)
Subject: Re: [CNU-Likelion/frontend-introduce-html] first commit (PR #9)

@jungbug approved this pull request.
건하님!
1주차 미션하시느라 수고 많으셨어요!
우선 코드 부분에서는 따로 수정할게 없는거 같아요 👍
시멘틱 태그들을 잘 사용해 주셨어요!
다만 코드에서 경로가 올바르지 않게 설정되어있는데 수정하시면 아마 버튼 누르면 잘 들어가질 거에요
또한 깃허브 사용에 관해서는 모르시면 저한테 연락 주시면 제가 도와드리겠습니다!
간단하게 push 하는 방법 알려드릴게요
git add . git commit -m "하고 싶은 말" git push origin thisisha
이런식으로 작성 해주시면 되요!
자세한 내용은 미션 진행 가이드 여기를 참고해주세요
In index.html:
> + <style> + /* 추가한 CSS 스타일 */ + body { + display: flex; + justify-content: center; + align-items: center; + flex-direction: column; /* 내용을 수직으로 정렬 */ + height: 100vh; /* 화면 전체 높이로 설정하여 수직 중앙 정렬 */ + margin: 0; /* body의 기본 margin 제거 */ + } + header { + background-color: lightgrey; /* 상단바 배경색을 회색으로 설정 */ + display: flex; + justify-content: space-between; + align-items: center; + padding: 10px; + width: 100%; /* 상단바 너비를 화면 전체로 설정 */ + position: fixed; /* 페이지를 스크롤해도 상단바가 고정되도록 설정 */ + top: 0; /* 상단바를 화면 상단에 고정 */ + } + .title { + flex-grow: 1; + } + nav { + margin-left: 20px; + } + .profile { + display: flex; + justify-content: center; + align-items: center; + text-align: center; /* 텍스트 중앙 정렬 */ + width: 80%; /* 내용의 너비를 설정 */ + margin-top: 100px; /* 상단바와의 간격 조정 */ + } + .profile-picture img { + width: 25%; /* 사진 크기를 4배 줄임 */ + height: auto; /* 비율 유지 */ + margin-right: 5px; /* 사진과 내용 사이 여백 */ + } + .profile-details { + width: 50%; /* 내용의 너비를 설정 */ + text-align: left; /* 자기소개 내용을 사진의 우측에 위치 */ + } + </style>
이런식으로 작성하는것도 파일에 이상은 없지만 따로 css 폴더를 만들어서 그 안에 css 파일들을 만들어서 관리하시면 나중에 유지보수 하기가 편해요!
In index.html:
> + <footer> + <!-- 푸터 내용을 추가하세요 --> + </footer>
footer는 사용하지 않으니까 지워도 될거 같아요!
In pages/goals.html:
> + <style> + /* 추가한 CSS 스타일 */ + body { + display: flex; + justify-content: center; + align-items: center; + flex-direction: column; /* 내용을 수직으로 정렬 */ + height: 100vh; /* 화면 전체 높이로 설정하여 수직 중앙 정렬 */ + margin: 0; /* body의 기본 margin 제거 */ + } + header { + background-color: lightgrey; /* 상단바 배경색을 회색으로 설정 */ + padding: 10px; + width: 100%; /* 상단바 너비를 화면 전체로 설정 */ + position: fixed; /* 페이지를 스크롤해도 상단바가 고정되도록 설정 */ + top: 0; /* 상단바를 화면 상단에 고정 */ + display: flex; + justify-content: space-between; + align-items: center; + } + .title { + flex-grow: 1; + } + nav { + margin-left: 20px; + display: flex; + } + nav a { + margin-right: 20px; + text-decoration: none; + color: black; + } + .blocks { + display: flex; /* 블록들을 가로로 나란히 배치 */ + justify-content: space-around; /* 블록들을 가로 방향으로 고르게 배치 */ + width: 100%; /* 블록들을 가로로 꽉 차게 설정 */ + margin-top: 50px; /* 블록과 상단바 사이 여백 설정 */ + } + .block { + width: 300px; /* 블록의 너비를 설정 */ + padding: 20px; /* 내용과 테두리 사이 여백 설정 */ + background-color: white; /* 블록 배경색을 흰색으로 설정 */ + text-align: center; /* 텍스트 중앙 정렬 */ + } + .block + .block { + margin-left: 20px; /* 두 번째 블록부터 왼쪽 여백 설정 */ + } + .block h2 { + background-color: darkgrey; /* 제목 배경색을 더 진한 회색으로 설정 */ + color: white; /* 제목 글자색을 흰색으로 설정 */ + padding: 10px; /* 제목의 위아래 여백 설정 */ + margin: 0; /* 제목의 기본 margin 제거 */ + } + .block p { + padding: 20px; /* 내용과 테두리 사이 여백 설정 */ + background-color: lightgrey; /* 내용 배경색을 더 연한 회색으로 설정 */ + color: black; /* 내용 글자색을 검은색으로 설정 */ + margin: 0; /* 내용의 기본 margin 제거 */ + } + .block-content { + background-color: lightgrey; /* 내용 배경색을 더 연한 회색으로 설정 */ + padding: 20px; /* 내용과 테두리 사이 여백 설정 */ + margin-top: 0px; /* 내용 위 여백 설정 */ + margin-bottom: 30px; /* 내용 아래 여백 설정 */ + text-align: center; /* 텍스트 중앙 정렬 */ + } + </style>
여기도 마찬가지로 css 폴더 안에 파일 만들어서 관리하면 좋을거 같아요!
In pages/goals.html:
> + <nav> + <a href="index.html">About me</a> + <a href="pages/hobby.html">Hobby</a> + <a href="pages/goals.html">Goals</a> + </nav>
여기부분에서 경로가 잘못된거 같아요!
<nav> <a href="../index.html">About me</a> <a href="hobby.html">Hobby</a> <a href="goals.html">Goals</a> </nav>
이런식으로 바꿔주시면 아마 잘 들어가질 거에요!
In pages/hobby.html:
> + <nav> + <a href="index.html">About me</a> + <a href="pages/hobby.html">Hobby</a> + <a href="pages/goals.html">Goals</a> + </nav>
여기 부분도 수정하시면 잘 들어가질거에요!
<nav> <a href="../index.html">About me</a> <a href="hobby.html">Hobby</a> <a href="goals.html">Goals</a> </nav>
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
